### PR TITLE
[ES] Update `Community/Project Loved`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by agatemosu** [comment]
y tal

File: wiki/Community/Project_Loved/es.md [hunk 1]
```diff
@@ -28,32 +23,29 @@ Los creadores de estos beatmaps pueden enviar una petición para eliminarlos de
 
 Existen unos criterios mínimos que deben cumplir los beatmaps que son nominados a Amados. Cada modo de juego tiene sus propios criterios específicos de nominación y prioridad, pero todos los modos de juego siguen las siguientes reglas:
```
Comment (agatemosu) [inline wiki/Community/Project_Loved/es.md:24]:
```suggestion
Existen unos criterios mínimos que deben cumplir los beatmaps que son nominados a amados. Cada modo de juego tiene sus propios criterios específicos de nominación y prioridad, pero todos los modos de juego siguen las siguientes reglas:
```

File: wiki/Community/Project_Loved/es.md [hunk 2]
```diff
@@ -1,8 +1,3 @@
----
-outdated_since: 5da0b97c91041623a03886001e26fae4d030765e
-outdated_translation: true
----
-
 # Project Loved
 
 **Project Loved** es la plataforma de votación que promueve beatmaps a la categoría [Amados](/wiki/Beatmap/Category#loved). Es coordinado principalmente por el [Project Loved Team](/wiki/People/Project_Loved_Team).
```
Comment (agatemosu) [inline wiki/Community/Project_Loved/es.md:3]:
```suggestion
**Project Loved** es la plataforma de votación que promueve beatmaps a la categoría [amados](/wiki/Beatmap/Category#loved). Es coordinado principalmente por el [Project Loved Team](/wiki/People/Project_Loved_Team).
```

File: wiki/Community/Project_Loved/es.md [hunk 3]
```diff
@@ -1,11 +1,6 @@
...
-outdated_since: 5da0b97c91041623a03886001e26fae4d030765e
-outdated_translation: true
----
-
 # Project Loved
 
-**Project Loved** es la plataforma de votación que promueve beatmaps a la categoría [Amados](/wiki/Beatmap/Category#loved). Es coordinado principalmente por el [Project Loved Team](/wiki/People/Project_Loved_Team).
+**Project Loved** es la plataforma de votación que promueve beatmaps a la categoría [amados](/wiki/Beatmap/Category#loved). Es coordinado principalmente por el [Project Loved Team](/wiki/People/Project_Loved_Team).
 
 ## Promoviendo un beatmap a Amados
```
Comment (agatemosu) [inline wiki/Community/Project_Loved/es.md:5]:
```suggestion
## Promoviendo un beatmap a amados
```

File: wiki/Community/Project_Loved/es.md [hunk 4]
```diff
@@ -1,13 +1,8 @@
...
-
 # Project Loved
 
-**Project Loved** es la plataforma de votación que promueve beatmaps a la categoría [Amados](/wiki/Beatmap/Category#loved). Es coordinado principalmente por el [Project Loved Team](/wiki/People/Project_Loved_Team).
+**Project Loved** es la plataforma de votación que promueve beatmaps a la categoría [amados](/wiki/Beatmap/Category#loved). Es coordinado principalmente por el [Project Loved Team](/wiki/People/Project_Loved_Team).
 
-## Promoviendo un beatmap a Amados
+## Promoviendo un beatmap a amados
 
 Los beatmaps son promovidos a la categoría Amados por un proceso de nominación y votación.
```
Comment (agatemosu) [inline wiki/Community/Project_Loved/es.md:7]:
```suggestion
Los beatmaps son promovidos a la categoría amados por un proceso de nominación y votación.
```

**Review by TicClick — APPROVED** [review]

**Pull request merged**